src/test/java/com/google/api/codegen/testsrc/common/library.proto

```diff
@@ -816,7 +816,7 @@ message GetBookFromAnywhereRequest {
   // The name of the book to retrieve.
   string name = 1 [
     (google.api.field_behavior) = REQUIRED,
-    (google.api.resource_reference) = "google.example.library.v1.Book"];
+    (google.api.resource_reference) = "google.example.library.v1.BookOneOf"];
```

Comment:
This matches what library_gapic.yaml says, and also tests (same) package names in `resource_reference`